### DISCUSSION TITLE
how to add custom button to sidebar

### DISCUSSION THREAD

**Discussion by kobe651jp** [excalidraw/excalidraw#5433] [Q&A]
I'm trying to save library to cloud storage and reload it.

It's possible to append button and handle the event?

![image](https://user-images.githubusercontent.com/44263972/178195313-c5af0b40-5b9f-4769-93a4-0697f2659c44.png)

**Comment by dwelle** [comment]
Not without forking. We're looking into how we can easily allow for these things, though. If you have ideas for an component-based API or something other than peppering every component with 10 different render props, share your thoughts!

**Reply by kobe651jp** [reply]
I've forked for add [japanese-handwrite ](https://github.com/kobe651jp/excalidraw-jpnHandWrite) font.
I'll try apply custom button to sidebar.
I'll let you know when it's done.
